addons/web/static/src/js/views/calendar/calendar_model.js

```diff
@@ -349,27 +349,27 @@ return AbstractModel.extend({
      * @returns {Object}
      */
     _getFullCalendarOptions: function () {
-        var $views;
-        var year = new Date().getFullYear();
-        if (!config.isMobile || (year.toString() != this.data.end_date.format('YYYY'))) {
-            $views = {
-                week: {
-                    columnFormat: 'ddd ' + time.strftime_to_moment_format(_t.database.parameters.date_format),
-                    titleFormat: time.strftime_to_moment_format(_t.database.parameters.time_format),
-                }
-            };
-        } else {
-            if (year.toString() == this.data.end_date.format('YYYY')) {
-                var currentDateTime = _t1.database.parameters.date_format;
-                currentDateTime.startsWith('%Y') ? currentDateTime = currentDateTime.replace(/%Y[\W]/i,' ') : currentDateTime = currentDateTime.replace(/[\W]%Y/i,' ');
-                $views = {
-                    week: {
-                        columnFormat: 'ddd ' + time.strftime_to_moment_format(currentDateTime),
-                        titleFormat: time.strftime_to_moment_format(_t.database.parameters.time_format),
-                    }
-                };
-            }
-        }
+        // var $views;
```

Comment:
@suh-odoo why, oh why, do you comment all these lines? That doesn't make much sense.